module/nntest/module.py

```diff
@@ -1047,6 +1054,9 @@ def get_lib_env_uoas(self, platform):
             x = r.get('deps',{}).get('library',{}).get('uoa','')
             if x:
                 uoas.append(x)
+        # Filter by specified lib uoas
```

Comment:
UOAs

Reply:
Fixed. And some other 'uoa's in comment too